### PR TITLE
T10 - sign in UI

### PR DESCRIPTION
**Pull request by idaosk**
this PR is for login UI only. 
login page : username and password , signin button

### CODE REVIEW

File: app/src/main/java/com/gathering/android/intro/IntroActivity.kt [hunk 1]
```diff
@@ -4,6 +4,7 @@ import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.gathering.android.R
+import com.gathering.android.SignInActivity
```
Comment (amirziarati) [inline app/src/main/java/com/gathering/android/intro/IntroActivity.kt:7]: unused import !

File: app/src/main/res/drawable/back_sign_up.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
     android:shape="rectangle">
     <gradient
         android:angle="90"
-        android:startColor="#F1B1BC"
```
Comment (amirziarati) [inline app/src/main/res/drawable/back_sign_up.xml:6]: lets not hardcode colours here. define colour in colour.xml file for it.

File: app/src/main/res/drawable/back_sign_up.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
-        android:endColor="#EFEBE0"
```
Comment (amirziarati) [inline app/src/main/res/drawable/back_sign_up.xml:7]: same as above

File: app/src/main/res/layout/activity_sign_up.xml [hunk 4]
```diff
@@ -87,7 +87,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:orientation="horizontal"
-        app:layout_constraintGuide_begin="300dp" />
+        app:layout_constraintGuide_begin="250dp" />
```
Comment (amirziarati) [inline app/src/main/res/layout/activity_sign_up.xml:90]: dont use magical numbers. add the value to dimen.xml file.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -9,4 +9,5 @@
     <string name="fragment_3_text">Invite guests and manage the event details!</string>
     <string name="signIn_button_text">Sign In</string>
     <string name="signUp_button_text">Sign Up</string>
+    <string name="str_login_page">Login</string>
```
Comment (amirziarati) [inline app/src/main/res/values/strings.xml:12]: unused string

**Review by amirziarati — APPROVED** [review]

File: app/src/main/res/layout/activity_sign_up.xml (hunk 4 above)
Comment (Animansoubi) [reply app/src/main/res/layout/activity_sign_up.xml:90]: Yes, it was mine. Sure I'll create a ticket and would try to fix it

**Review by Animansoubi — COMMENTED** [review]
To my knowledge, everything looks good :)

**Pull request merged**